commands/service_compile.go

```diff
@@ -244,6 +245,13 @@ func (s *arduinoCoreServerImpl) Compile(req *rpc.CompileRequest, stream rpc.Ardu
 			Message: &rpc.CompileResponse_Progress{Progress: p},
 		})
 	}
+	var verbosity logger.Verbosity = logger.VerbosityNormal
+	if req.GetQuiet() {
+		verbosity = logger.VerbosityQuiet
+	}
+	if req.GetVerbose() {
+		verbosity = logger.VerbosityVerbose
+	}
```

Comment:
If we pass both flags what is the expected behaviour?
1. Should we make them mutually exclusive?
2. Should the verbose win as it is now?

I have a slightly prefernce in option n2, as it will help dev that are using the gRPC interface to better track the problem in case they are providing both flag by mistake. 
In the cli interface we can use cobra to enforce the mutually exclusiveness of the -q and -v flags, so we don't have to bother to handle this errors as they are prevented in the first place.

_In an ideal world pre 1.0.0, we would have had an enum property containing the possible verbosity level, but it's not possible as it would be a breaking change. :cry: _

Reply:
> I have a slightly prefernce in option n2, as it will help dev that are using the gRPC interface to better track the problem in case they are providing both flag by mistake.

Let's keep `verbose` overriding `quiet`.

> In an ideal world pre 1.0.0, we would have had an enum property containing the possible verbosity level, but it's not possible as it would be a breaking change. 

Yep, unfortunately, the inconsistency of having both `verbose` and `quiet` in the same RPC call slipped through the cracks... (and it's even more unfortunate because `quiet` was unused and easy to spot 🤦🏼).